backend/fms_core/resources/project_link_sample.py

```diff
@@ -0,0 +1,104 @@
+import reversion
+from import_export.fields import Field
+from import_export.widgets import ForeignKeyWidget
+from django.core.exceptions import ValidationError, ObjectDoesNotExist
+from ._generic import GenericResource
+from ._utils import skip_rows, add_columns_to_preview
+from ..models import Project, Sample, Container, SampleByProject
+from ..utils import get_normalized_str
+
+
+__all__ = ["ProjectLinkSampleResource"]
+
+ADD_ACTION = "ADD"
+REMOVE_ACTION = "REMOVE"
+
+class ProjectLinkSampleResource(GenericResource):
+    action = Field(column_name='Action')
+    project_name = Field(column_name='Project Name') 
+    sample_name = Field(column_name='Sample Name')
+    barcode = Field(column_name='Sample Container Barcode',
+                     widget=ForeignKeyWidget(Container, 'barcode'))
+    coordinates = Field(column_name='Sample Container Coord')
+
+    class Meta:
+        model = SampleByProject
```

Comment:
Don't forget the import_id_fields and fields such as in: 
        import_id_fields = ()
        fields = (
        )

Otherwise you get a bunch of internal fields that are not relevant to the user:
![image](https://user-images.githubusercontent.com/1617741/133154738-7128d20e-cbcd-426d-a191-42b52de52509.png)